variants/NUCLEO_L476RG/variant.cpp

```diff
@@ -125,9 +125,10 @@ WEAK void SystemClock_Config(void)
   RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
 
   /* MSI is enabled after System reset, activate PLL with MSI as source */
-  RCC_OscInitStruct.OscillatorType = RCC_OSCILLATORTYPE_MSI;
+  RCC_OscInitStruct.OscillatorType = RCC_OSCILLATORTYPE_MSI|RCC_OSCILLATORTYPE_HSI;
```

Comment:
so that means we will always have HSI and MSI running ... can't we keep HSI only as the purpose of this PR is to save power  ?

Reply:
We'll look the impact of keep only HSI. But I think we could.